scio-google-cloud-platform/src/it/scala/com/spotify/scio/bigquery/BigQueryDynamicTableIOIT.scala

```diff
@@ -44,28 +46,30 @@ object DynamicBigQueryIT {
   def newRecord(x: Int): Record = Record(x, x.toString)
 }
 
-class DynamicBigQueryIT extends AnyFlatSpec with Matchers {
-  import DynamicBigQueryIT._
+class BigQueryDynamicTableIOIT extends AnyFlatSpec with Matchers {
 
-  private val bq = BigQuery.defaultInstance()
+  import BigQueryDynamicTableIOIT._
+  import ItUtils.project
 
-  private val options = PipelineOptionsFactory
-    .fromArgs(s"--project=$projectId", "--tempLocation=gs://data-integration-test-eu/temp")
+  private val bq = BigQuery.defaultInstance()
+  private val options: PipelineOptions = PipelineOptionsFactory
+    .fromArgs(s"--project=$project", s"--tempLocation=$tempLocation")
     .create()
 
-  "Dynamic BigQuery" should "support typed output" in {
+  it should "support typed output" in {
     val prefix = UUID.randomUUID().toString.replaceAll("-", "")
     val sc = ScioContext(options)
-    // Assigning the value to coll to workaround a bug in scalac...
-    sc.parallelize(1 to 10)
+
+    sc.parallelize(1 to 3)
```

Comment:
why decrease to 3?

Reply:
the results for 10 / 3 are the same, I reduced it as I was running tests locally - bumped this back to 10.